agent/src/index.test.ts

```diff
@@ -93,7 +111,19 @@ describe.each([
     it('initializes properly', async () => {
         const serverInfo = await client.handshake(clientInfo)
         assert.deepStrictEqual(serverInfo.name, 'cody-agent', 'Agent should be cody-agent')
-        assert.deepStrictEqual(serverInfo.codyEnabled, true, 'Cody should be enabled')
+        assert.deepStrictEqual(
+            serverInfo.codyEnabled,
+            name !== 'NotConfigured',
+            'Cody should be enabled when configured'
+        )
```

Comment:
```suggestion
        expect(serverInfo.codyEnabled).not.toBe('NotConfigured')
```

Might be a bit easier to read + nicer visualized in a failed run if we use vitest assertions instead of the built-in `assert` lib

Reply:
The correct translation is `expect(serverInfo.codyEnabled).toBe(name !== 'Not Configured')`. Right now, we only use `assert` in this file so I think it's better to do a wholesale refactoring in the future if we prefer `expect` (which I personally don't like using so much)